packages/flutter_test/pubspec.yaml

```diff
@@ -29,6 +29,9 @@ dependencies:
   # Used by globalToLocal et al.
   vector_math: 2.1.4
 
+  # Used by testWidgets.
+  leak_tracker: 2.0.1
```

Comment:
This is what will become a bit challenging that @goderbauer is referring to.

One option would be to start with this as a `dev_dependency` until it's been vetted a bit more with the framework tests.

At the same time, we should have a good cycle of time to let this get ready before another stable...

Reply:
To evaluate this with the framework tests we'd ideally keep it as a `dev_dependency` on the framework. Not super sure how we can do that with how things are currently setup, though. May need some extra thought.